src/Instance/HoldingsList/Holding/HoldingButtonsGroup.js

```diff
@@ -42,7 +47,8 @@ const HoldingButtonsGroup = ({
       <Button
         id={`clickable-view-holdings-${holding.id}`}
         data-test-view-holdings
-        onClick={onViewHolding}
+        onClick={() => updateAffiliation(stripes.okapi, tenantId, onViewHolding)}
+        disabled={isViewHoldingsDisabled}
```

Comment:
> I think we shouldn't display elements if a user doesn't have permission. I guess it's a FOLIO pattern.

Yep, that's the FOLIO UX I'm familiar with. I will follow up with the PO here as I see the story was written specifically with disabled elements, and I've seen similar stories in ui-users. We need to figure out if we are purposely abandoning that pattern or if this was just a goof.

Reply:
@zburke and @Dmytro-Melnyshyn This is a requirement for this story, but we discussed with PO that suppressing these buttons from a user instead of disabling them will make the page kinda less noisy and will free up more space. We decided to go with the disabled buttons in the main release and after further testing and feedback from users, we might end up hiding these buttons.